pep-0622.rst

```diff
@@ -582,7 +582,10 @@ Sequence Patterns
 
 Simplified syntax::
 
-  sequence_pattern: '[' [values_pattern] ']' | '(' [values_pattern] ')'
+  sequence_pattern:
+      | '[' [values_pattern] ']' 
+      | '(' [value_pattern ',' [values pattern]] ')' 
+      | value_pattern ',' [values pattern]
```

Comment:
I'd drop this line (see previous exchange).

Reply:
dropped and added a note